src/app/shell/provider/provider-config/provider-config.component.scss

```diff
@@ -1,21 +1,387 @@
-form{
+* {
```

Comment:
margin and padding should be reset in base styles for the entire app.

changing the default box sizing model for all elements may cause problems with supporting this component

is it really necessary to set border-box for every single element in the component?